examples/tutorials/lighting_tutorial.py

```diff
@@ -64,7 +60,7 @@ def make_configuration():
     backend_cfg.enable_physics = True
 
     # agent configuration
-    sensor_cfg = habitat_sim.SensorSpec()
+    sensor_cfg = habitat_sim.CameraSensorSpec()
     sensor_cfg.resolution = [1080, 960]
     agent_cfg = habitat_sim.agent.AgentConfiguration()
     agent_cfg.sensor_specifications = [sensor_cfg]
```

Comment:
Example of a single CameraSensorSpec initialization

Reply:
Now I am starting to see how a SensorFactory could be useful here.